docs/_interface/merge.md

```diff
@@ -3,36 +3,32 @@ title: The Merge
-change to how Ethereum operates and it will require node operators to run a consensus client in addition to
-Geth.
-
-## How will Geth change?
-
-Geth is an execution client. Historically, an execution client alone has been enough to run a full Ethereum node.
-However, when The Merge happens, Geth will not be able to track the Ethereum chain on its own. Instead, it will need to 
-be coupled to another piece of software called a ["consensus client"][con-client-link]. In this configuration, 
-the execution client will be responsible for transaction handling, transaction gossip, state management and
-the Ethereum Virtual Machine (EVM). However, Geth will no longer be responsible for block building, block
-gossiping or handling consensus logic. These will be in the remit of the consensus client.
-
-For Geth users that intend to continue to run full nodes after The Merge, it is sensible to start running
-a consensus client in advance, so that The Merge can happen smoothly. There are five consensus clients available, all
-of which connect to Geth in the same way. Instructions for this are available on the 
-[Consensus Clients page](/docs/interface/consensus-clients).
+The Merge was probably the most significant upgrade to Ethereum since Homestead. This page explains what The Merge was
+and how it affected Geth users.
+
+## What was The Merge
+
+[The Merge](https://ethereum.org/en/upgrades/merge/) was an upgrade to Ethereum that swapped out its original
+proof-of-work (PoW) based consensus mechanism for a proof-of-stake based (PoS) mechanism. The term "Merge" refers
+to the fact that the initial phase of the transition involved launching a PoS chain in parallel with the
+PoW Ethereum Mainnet. That PoS chain, known as the Beacon Chain, was been executing the PoS consensus logic but 
+without including any real Ethereum transaction data. The Merge was the moment when Ethereum's blockchain 
+and the Beacon Chain joined together to form one unified chain. At the moment of The Merge, execution clients 
+switched off their proof-of-work and block gossiping functions and handed responsibility for all consensus and 
+fork choice logic over to consensus clients. This was a profound change to how Ethereum operates. Now that The Merge
+is done, node operators are required to run a consensus client in addition to an execution client such as Geth.
+
+## How did Geth change?
+
+Geth is an execution client. Historically, running an execution client alone was enough to turn a computer 
+into a full Ethereum node. However, since The Merge, Geth has not been able to track the Ethereum chain on 
+its own. Instead, it needs to be coupled to another piece of software called a ["consensus client"][con-client-link]. 
+The execution client is responsible for transaction handling, transaction gossip, state management and
+the Ethereum Virtual Machine (EVM). However, Geth is no longer responsible for block building, block gossiping or 
```

Comment:
I think it may be a bit confusing to readers. Geth is still responsible for generate a building with pending transactions included. However it's not responsible for propose a block.

In this term, maybe Geth is still responsible for block building(block builder) but not a block proposer.

Reply:
yea ok - I guess what I mean is that Geth creates a component of the block, not the complete block.

```suggestion
the Ethereum Virtual Machine (EVM). However, Geth is no longer responsible for block proposals or 
```